Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -110,7 +110,9 @@ HackStudioWidget::HackStudioWidget(DeprecatedString path_to_project)
 
     m_editors_splitter = m_right_hand_stack->add<GUI::VerticalSplitter>();
     m_editors_splitter->layout()->set_margins({ 3, 0, 0 });
-    add_new_editor_tab_widget(*m_editors_splitter);
+
+    // FIXME: Propagate errors instead of allowing failure
+    MUST(add_new_editor_tab_widget(*m_editors_splitter));
```

Comment:
Instead of using `MUST()` for FIXMEs, please use ErrorOr's `release_value_but_fixme_should_propagate_errors()` method, like so:

```suggestion
    add_new_editor_tab_widget(*m_editors_splitter).release_value_but_fixme_should_propagate_errors();
```
It's long and awkward, but that's kind of the point. :^)